### PR TITLE
Feature/oauth2 permission grant delta v1

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

File: concepts/changelog.md [hunk 1]
```diff
@@ -96,6 +96,11 @@ For a summary of the value of these API changes, as well as recent tools, compon
 | :-------------- | :---------- | :--------------------------------------- |
 | Addition | v1.0 | Introduced the Shifts APIs, including [shift](/graph/api/resources/shift?view=graph-rest-1.0), [workforceintegration](/graph/api/resources/workforceintegration?view=graph-rest-1.0), and associated methods.|
 
+### Track changes
+
+| **Change type** | **Version** | **Description**                          |
+| :-------------- | :---------- | :--------------------------------------- |
+| Addition        | v1.0        | Added [delta query](delta-query-overview.md) capability for the following [oauth2PermissionGrant](/graph/api/oauth2Permissiongrant-delta) |
```
Comment (baywet) [inline concepts/changelog.md:103]:
```suggestion
| Addition        | v1.0        | Added [delta query](delta-query-overview.md) capability for [oauth2PermissionGrant](/graph/api/oauth2Permissiongrant-delta) |
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+author: "psignoret"
+ms.prod: "microsoft-identity-platform"
+doc_type: apiPageType
+---
+
+# oauth2permissiongrant: delta
+
+Namespace: microsoft.graph
+
+Get newly created, updated, or deleted **oauth2permissiongrant** objects without performing a full read of the entire resource collection. For details, see [Using delta query](/graph/delta-query-overview).
```
Comment (psignoret) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:14]: Could we link to the entity doc here?

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Content-type: application/json
+
+{
+  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#oauth2permissiongrants",
+  "@odata.nextLink":"https://graph.microsoft.com/v1.0/oauth2permissiongrants/delta?$skiptoken=pqwSUjGYvb3jQpbwVAwEL7yuI3dU1LecfkkfLPtnIjsXoYQp_dpA3cNJWc",
+  "value": [
+    {
+      "clientId": "22a3c970-8ad4-4120-8127-300837f87f2c",
+      "consentType": "Principal",
+      "expiryTime": "2017-08-13T21:41:23.3929007Z",
```
Comment (psignoret) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:108]: `startTime` and `expiryTime` aren't in v1.0

Comment (baywet) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:108]:
```suggestion
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      "principalId": "c2e8df37-c6a7-4d88-89b1-feb4f1fda7c5",
+      "resourceId": "98dc9d95-49b6-405a-b3c0-834e969a708b",
+      "scope": "User.Read Directory.AccessAsUser.All",
+      "startTime": "0001-01-01T00:00:00Z",
```
Comment (baywet) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:112]:
```suggestion
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 2 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:14]:
```suggestion
Get newly created, updated, or deleted **[oauth2permissiongrant](../resources/oauth2permissiongrant.md)** objects without performing a full read of the entire resource collection. For details, see [Using delta query](/graph/delta-query-overview).
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 3 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md]: fixed

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+author: "psignoret"
+ms.prod: "microsoft-identity-platform"
+doc_type: apiPageType
+---
+
+# oauth2permissiongrant: delta
+
+Namespace: microsoft.graph
+
+Get newly created, updated, or deleted **[oauth2permissiongrant](../resources/oauth2permissiongrant.md)** objects without performing a full read of the entire resource collection. For details, see [Using delta query](/graph/delta-query-overview).
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:14]: No need to bold the link to the resource. Remove the asterisks.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | Directory.Read.All, Directory.ReadWrite.All, Directory.AccessAsUser.All    |
+|Delegated (personal Microsoft account) | Not supported.    |
+|oauth2permissiongrant | Directory.Read.All, Directory.ReadWrite.All |
+
+## HTTP request
+
+To begin tracking changes, you make a request including the delta function on the **oauth2permissiongrant** resource.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:29]: This sentence isn't needed.

Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:29]:
```suggestion
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 5 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:14]:
```suggestion
Get newly created, updated, or deleted [oauth2permissiongrant](../resources/oauth2permissiongrant.md) objects without performing a full read of the entire resource collection. For details, see [Using delta query](/graph/delta-query-overview).
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+GET /oauth2permissiongrants/delta
+```
+
+## Query parameters
+
+Tracking changes incurs a round of one or more **delta** function calls. If you use any query parameter 
+(other than `$deltatoken` and `$skiptoken`), you must specify 
+it in the initial **delta** request. Microsoft Graph automatically encodes any specified parameters 
+into the token portion of the `nextLink` or `deltaLink` URL provided in the response. 
+You only need to specify any query parameters once up front.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:41]: You only specify query parameters once.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+| Query parameter	   | Type	|Description|
+|:---------------|:--------|:----------|
+| $deltatoken | string | A [state token](/graph/delta-query-overview) returned in the `deltaLink` URL of the previous **delta** function call for the same resource collection, indicating the completion of that round of change tracking. Save and apply the entire `deltaLink` URL, including this token, in the first request of the next round of change tracking for that collection.|
+| $skiptoken | string | A [state token](/graph/delta-query-overview) returned in the `nextLink` URL of the previous **delta** function call, indicating that there are further changes to be tracked in the same resource collection. |
+
+### Optional query parameters
+
+This method supports OData query parameters to help customize the response.
+
+- You can use a `$select` query parameter as in any GET request to specify only the properties your need for best performance. The **id** property is always returned.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:54]: You can use the `$select` query parameter in a GET request to specify only the properties that you need for best performance. The **id** property is always returned.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+- There is limited support for `$filter`:
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:55]: This line of text isn't needed.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 7 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:41]:
```suggestion
You only need to specify query parameters once. 
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 8 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:54]:
```suggestion
- You can use a `$select` query parameter similarly to any GET request to specify only the properties your need for best performance. The **id** property is always returned.
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 9 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:55]:
```suggestion
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+  * The only supported `$filter` expression is for tracking changes for specific resources, by their ID:  `$filter=id+eq+{value}` or `$filter=id+eq+{value1}+or+id+eq+{value2}`. The number of IDs you can specify is limited by the maximum URL length.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:55]: The `$filter` query parameter can only be used to track changes on specific resources by using the resource ID. For example,  `$filter=id+eq+{value}` or `$filter=id+eq+{value1}+or+id+eq+{value2}`.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+
+
+## Request headers
+| Name       | Description|
+|:---------------|:----------|
+| Authorization  | Bearer &lt;token&gt;. Required.|
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:61]: Bearer {token}

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+
+## Request body
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and an [oauth2permissiongrant](../resources/oauth2permissiongrant.md) collection object in the response body. The response also includes a `nextLink` URL or a `deltaLink` URL.
+
+- If a `nextLink` URL is returned, there are additional pages of data to be retrieved in the session. The **oauth2permissiongrant** continues making requests using the `nextLink` URL until a `deltaLink` URL is included in the response.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:70]: If a `nextLink` URL is returned, additional pages of data can be retrieved in the session.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 11 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:61]:
```suggestion
| Authorization  | Bearer {token}. Required.|
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 10 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:55]:
```suggestion
- The `$filter` query parameter can only be used to track changes on specific resources by using the resource ID. For example, `$filter=id+eq+{value}` or `$filter=id+eq+{value1}+or+id+eq+{value2}`.
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+- If a `deltaLink` URL is returned, there is no more data about the existing state of the resource to be returned. Persist and use the `deltaLink` URL to learn about changes to the resource in the future.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:71]: If a `deltaLink` URL is returned, no more data about the resource is returned.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 12 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:70]:
```suggestion
- If a `nextLink` URL is returned, additional pages of data can be retrieved in the session. The **oauth2permissiongrant** continues making requests using the `nextLink` URL until a `deltaLink` URL is included in the response.
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 13 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:71]:
```suggestion
- If a `deltaLink` URL is returned, no more data about the resource is returned. Persist and use the `deltaLink` URL to learn about changes to the resource in the future.
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 14]
```diff
@@ -0,0 +1,127 @@
...
+
+## Query parameters
+
+Tracking changes incurs a round of one or more **delta** function calls. If you use any query parameter 
+(other than `$deltatoken` and `$skiptoken`), you must specify 
+it in the initial **delta** request. Microsoft Graph automatically encodes any specified parameters 
+into the token portion of the `nextLink` or `deltaLink` URL provided in the response. 
+You only need to specify query parameters once. 
+In subsequent requests, copy and apply the `nextLink` or `deltaLink` URL from the previous response. That URL already 
+includes the encoded parameters.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:43]: The URL includes the encoded parameters.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+| Query parameter	   | Type	|Description|
+|:---------------|:--------|:----------|
+| $deltatoken | string | A [state token](/graph/delta-query-overview) returned in the `deltaLink` URL of the previous **delta** function call for the same resource collection, indicating the completion of that round of change tracking. Save and apply the entire `deltaLink` URL, including this token, in the first request of the next round of change tracking for that collection.|
+| $skiptoken | string | A [state token](/graph/delta-query-overview) returned in the `nextLink` URL of the previous **delta** function call, indicating that there are further changes to be tracked in the same resource collection. |
+
+### Optional query parameters
+
+This method supports OData query parameters to help customize the response.
+
+- You can use a `$select` query parameter similarly to any GET request to specify only the properties your need for best performance. The **id** property is always returned.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:54]: You can use the `$select` query parameter to specify only the properties that you need for best performance.

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+## Query parameters
+
+Tracking changes incurs a round of one or more **delta** function calls. If you use any query parameter 
+(other than `$deltatoken` and `$skiptoken`), you must specify 
+it in the initial **delta** request. Microsoft Graph automatically encodes any specified parameters 
+into the token portion of the `nextLink` or `deltaLink` URL provided in the response. 
+You only need to specify query parameters once. 
+In subsequent requests, copy and apply the `nextLink` or `deltaLink` URL from the previous response. That URL already
```
Comment (baywet) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:42]:
```suggestion
In subsequent requests, copy and apply the `nextLink` or `deltaLink` URL from the previous response. The URL 
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md (hunk 15 above)
Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:54]:
```suggestion
- You can use a `$select` query parameter to specify only the properties your need for best performance. The **id** property is always returned.
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 17]
```diff
@@ -0,0 +1,127 @@
...
+| Query parameter	   | Type	|Description|
+|:---------------|:--------|:----------|
+| $deltatoken | string | A [state token](/graph/delta-query-overview) returned in the `deltaLink` URL of the previous **delta** function call for the same resource collection, indicating the completion of that round of change tracking. Save and apply the entire `deltaLink` URL, including this token, in the first request of the next round of change tracking for that collection.|
+| $skiptoken | string | A [state token](/graph/delta-query-overview) returned in the `nextLink` URL of the previous **delta** function call, indicating that there are further changes to be tracked in the same resource collection. |
+
+### Optional query parameters
+
+This method supports OData query parameters to help customize the response.
+
+- You can use a `$select` query parameter to specify only the properties your need for best performance. The **id** property is always returned.
```
Comment (davidmu1) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:54]: your should be you

Comment (baywet) [reply api-reference/v1.0/api/oauth2permissiongrant-delta.md:54]:
```suggestion
- You can use a `$select` query parameter to specify only the properties you need for best performance. The **id** property is always returned.
```

File: concepts/changelog.md [hunk 18]
```diff
@@ -127,6 +127,11 @@ For a summary of the value of these API changes, as well as recent tools, compon
 [TeamsAppInstallation.ReadWriteForTeam](/graph/permissions-reference#Teams-app-installation-permissions), [TeamsAppInstallation.ReadForTeam.All](/graph/permissions-reference#Teams-app-installation-permissions), 
 [TeamsAppInstallation.ReadWriteForTeam.All](/graph/permissions-reference#Teams-app-installation-permissions). |
 
+### Track changes
```
Comment (Lauragra) [inline concepts/changelog.md:130]: The H3s in the changelog reflect our IA/docs taxonomy. We don't represent Track changes in our taxonomy, so I think the H3 here should be "Identity and access | Identity and sign-in". Also, we need a period at the end of the sentence. :)

File: concepts/changelog.md [hunk 19]
```diff
@@ -107,6 +107,7 @@ For a summary of the value of these API changes, as well as recent tools, compon
 | Addition | beta | Added the [entitlementManagementSettings](/graph/api/resources/entitlementmanagementsettings?view=graph-rest-beta) entity. |
 | Addition | v1.0 | Added the [identitySecurityDefaultsEnforcementPolicy API](/graph/api/resources/identitysecuritydefaultsenforcementpolicy?view=graph-rest-v1.0), which represents Azure Active Directory security defaults policy.|
 | Addition        | v1.0        | Added [delta query](delta-query-overview.md) capability to [servicePrincipal](/graph/api/serviceprincipal-delta). |
+| Addition        | v1.0        | Added [delta query](delta-query-overview.md) capability for [oauth2PermissionGrant](/graph/api/oauth2Permissiongrant-delta) |
```
Comment (baywet) [inline concepts/changelog.md:110]:
```suggestion
| Addition        | v1.0        | Added [delta query](delta-query-overview.md) capability for [oauth2PermissionGrant](/graph/api/oauth2Permissiongrant-delta). |
```

File: concepts/changelog.md (hunk 18 above)
Comment (baywet) [reply concepts/changelog.md]: I moved it to the identity and access table, right under the service principal entry. Thanks!

**Pull request merged**